services/services.go

```diff
@@ -5,31 +5,33 @@ type Service string
 
 // The list of registered services that publish scopes and/or roles.
 const (
-	Amp              Service = "amp"
-	CodyGateway      Service = "cody_gateway"
-	Dotcom           Service = "dotcom"
-	SAMS             Service = "sams"
-	TelemetryGateway Service = "telemetry_gateway"
-	EnterprisePortal Service = "enterprise_portal"
-	MailGatekeeper   Service = "mail_gatekeeper"
-	Workspaces       Service = "workspaces"
-	SSC              Service = "ssc"
-	Analytics        Service = "analytics"
-	CloudAPI         Service = "cloud_api"
+	Amp               Service = "amp"
+	CodyGateway       Service = "cody_gateway"
+	Dotcom            Service = "dotcom"
+	SAMS              Service = "sams"
+	TelemetryGateway  Service = "telemetry_gateway"
+	EnterprisePortal  Service = "enterprise_portal"
+	MailGatekeeper    Service = "mail_gatekeeper"
+	Workspaces        Service = "workspaces"
+	SSC               Service = "ssc"
+	Analytics         Service = "analytics"
+	CloudAPI          Service = "cloud_api"
+	ReleaseOperations Service = "releaseoperations"
```

Comment:
Would `release_operations` be more in line with the existing services?

Reply:
This matches the other release services, which started with `releaseregistry`. I know it's kind of against the format for everything else, but at this point we kind of have our own standard on the release stuff for better or worse.